.github/workflows/pages.yml

```diff
@@ -17,11 +17,16 @@ jobs:
           persist-credentials: false
           fetch-depth: 0
 
+      - name: Setup node env 🏗
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14
+
       - name: Install dependencies 👨🏻‍💻
-        run: cd web & yarn
+        run: cd web && yarn
 
       - name: Build 🔧
```

Comment:
We need to set `VUE_APP_ATLAS_API_ADDR = "https://api.atlas.cosmos.network/api/v1"` env var here when we build. This is how Atlas knows what API to reach out to.

Reply:
added, thanks